src/app/error-display/error-display.pug

```diff
@@ -1,7 +1,7 @@
 .error-display.col-xs-12
   i.mdi.mdi-alert-circle
   .message(ng-if="ctrl.error" ng-switch="ctrl.error.code")
-    span(ng-switch-when="400") {{ 'Unable to update the event from your request' | translate }}
+    span(ng-switch-when="400") {{ 'Unable to update the event' | translate }}
```

Comment:
So, it was not correct before?

Reply:
@tuanlc I didn't find any translation anywhere else matching that message, but there is one for 'Unable to update the event'